backend/qemu.pm

```diff
@@ -490,6 +490,7 @@ sub start_qemu {
     bmwqemu::save_vars();                 # update variables
 
     mkpath($basedir);
+    runcmd('/usr/bin/chattr', '+C', $basedir);
```

Comment:
we do not even use journaling file systems for the pool - and as such it will fail like
```
chattr: Inappropriate ioctl for device while reading flags on /dev/shm/testfile
```